src/rest_api.py

```diff
@@ -13,11 +14,14 @@
 from repo_dependency_creator import RepoDependencyCreator
 from notification.user_notification import UserNotification
 from fabric8a_auth.errors import AuthError
+import sentry_sdk
 
 
 app = Flask(__name__)
 CORS(app)
 
+sentry_sdk.init(os.environ.get("SENTRY_DSN"))
```

Comment:
@animuk - why is it not using raven and sentry? Is there any reason to use sentry_sdk?

Reply:
There are two reasons @miteshvp 
1. Using sentry_sdk is latest implementation than using raven
2. in the rest_api.py files, we do not see any logger being instantiated and raven and sentry object requires logger here.

Please let me know your commnets

Reply:
@animuk - then we should change the implementation everywhere to be consistent. WDYT?

Reply:
sure @miteshvp. we can change it in phases. can it be approved for now?

Reply:
@animuk how critical it is to do the integration to be done this way? I mean, will the raven way of integration be replaced eventually by the sentry_sdk?

Reply:
@samuzzal-choudhury please check the link - https://docs.sentry.io/platforms/python/logging/
sentry_sdk.init() already integrates with the logging module. It is equivalent to this explicit configuration. So, I guess raven integration still works where we are manually setting the logger.

Reply:
@animuk I am fine with the gemini implementation. Let's have a discussion about replicating the sentry_sdk implementation to other places.